src/main/java/user/createCorporatePage/exceptions/PageCreationFailed.java

```diff
@@ -0,0 +1,5 @@
+package user.createCorporatePage.exceptions;
+
+public class PageCreationFailed extends RuntimeException{
```

Comment:
It's good that you put this into an exceptions folder, I would still advise you to rename this class to PageCreationFailedException, it's just convention to have Exception in any exception class that you make that extends an exception class.